js/main.js

```diff
@@ -47,6 +51,8 @@ function getUrlAndOpenMainWindow() {
         let title = 'Error loading configuration';
         electron.dialog.showErrorBox(title, title + ': ' + err);            
     });
+
+    applyContextMenu();
```

Comment:
can we instead move this into windowMgr.js code and put in createMainWindow and createChildWindow and then pass window object in via opts parameter...e.g., applyContextMenu({ window: mainWindow});  and plyContextMenu({ window: childWindow});